index.js

```diff
@@ -7,5 +7,43 @@ new cursoreffects.springyEmojiCursor({ element: document.querySelector("#springs
 new cursoreffects.emojiCursor({ element: document.querySelector("#emoji") })
 new cursoreffects.bubbleCursor({ element: document.querySelector("#bubbles") })
 new cursoreffects.snowflakeCursor({ element: document.querySelector("#snowflake") })
+new cursoreffects.characterCursor({ 
```

Comment:
how can we use these on chromebook

Reply:
Sorry are you implying that this code doesn't work on Chromebook? Can you elaborate further